lua/lint/linters/rubocop.lua

```diff
@@ -0,0 +1,70 @@
+local severities = {
+  error = vim.lsp.protocol.DiagnosticSeverity.Error,
+  fatal = vim.lsp.protocol.DiagnosticSeverity.Error,
+  warning = vim.lsp.protocol.DiagnosticSeverity.Warning,
+  refactor = vim.lsp.protocol.DiagnosticSeverity.Warning,
+  convention = vim.lsp.protocol.DiagnosticSeverity.Warning,
+  info = vim.lsp.protocol.DiagnosticSeverity.Warning,
```

Comment:
The choice for this mapping matches that ALE does. We could map it to other LSP severities such as Hint, but this may result in diagnostics not showing up when you have those filtered out. For example, I myself filter out hints and info messages because most language servers just produce a ton of noise using those diagnostic levels.

This behaviour also matches the rubocop CLI, in that the CLI doesn't care about the severity and treats all of them as offenses.